templates/navbar.tt2

```diff
@@ -1,7 +1,7 @@
     <nav class='navbar navbar-expand-lg'>
       <a class='navbar-brand' href='https://opensuse.org' title='openSUSE'>
         [% IF theme.logo %]
-        <img class="mr-2" src="[% theme.logo %]" alt="[% theme.vendor %]"/>
+        <img class="mr-2" height="30" src="[% theme.logo %]" alt="[% theme.vendor %]"/>
```

Comment:
The fixed height might be a problem in other themes - either make it configurable or better use a css class or id which we can define in the theme